From: Oleksij Rempel <o.rempel@pengutronix.de>
Date: Tue, 14 Nov 2023 11:35:04 +0100
Subject: [PATCH] arm64: dts: freescale: fixes

Use rgmii-rxid instead of rgmii-txid. It has no effect on correct kernel
but rxid is probably proper value in this case.

Reduce regulator-uv-less-critical-window-ms time. So, we are able to
store reset reason in the RTC clock.

Signed-off-by: Oleksij Rempel <o.rempel@pengutronix.de>
---
 arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi b/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
index c1a2dee4e9a3..8ca2365847e4 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
@@ -120,7 +120,7 @@ reg_24v: regulator-24v {
 		pinctrl-0 = <&pinctrl_reg24v>;
 		interrupts-extended = <&gpio4 23 IRQ_TYPE_EDGE_FALLING>;
 		system-critical-regulator;
-		regulator-uv-less-critical-window-ms = <50>;
+		regulator-uv-less-critical-window-ms = <10>;
 	};
 
 	reg_can2rs: regulator-can2rs {
@@ -228,7 +228,7 @@ adc: adc@0 {
 &eqos {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_eqos>;
-	phy-mode = "rgmii-txid";
+	phy-mode = "rgmii-rxid";
 	status = "okay";
 
 	fixed-link {
